### DISCUSSION TITLE
Null topo_ra.grd and trans.dat files are being created

### DISCUSSION THREAD

**Discussion by vis2hnu1** [gmtsar/gmtsar#828] [Q&A]
Dear @Xiaohua-Eric-Xu Sir,

Currently, I'm working on generating the InSAR Time Series analysis by using sentinel-1 data. Sir, I'm facing the error after generating the interferogram pairs exactly. My error is after running "intf_tops.csh one.in batch_tops.config" here it is creating null  "trans.dat" and "topo_ra.grd" files. For better understanding, I'm attaching the log file which was generated at the time of running this command. Please help me to rectify this error sir.
[itp.log](https://github.com/gmtsar/gmtsar/files/12879882/itp.log)

**Comment by Xiaohua-Eric-Xu** [comment]
The failure stated at the beginning where it says "can't open  S1_20180907_ALL_F2.LED ". Please delete the file inside your topo directory and run the processing again.

**Reply by vis2hnu1** [reply]
Thank you for you help sir, It is rectified.
